src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -24,7 +24,8 @@ internal enum Stamp
 
         private readonly ArrayList m_list = new ArrayList();
 
-        private DateTime m_timeStamp = DateTime.MinValue; // Do not rename (binary serialization)
+        private int m_version; // Do not rename (binary serialization)
+        private DateTime m_TimeStamp = DateTime.MinValue; // Do not rename (binary serialization)
         private bool m_has_other_versions; // Do not rename (binary serialization)
```

Comment:
About which value are we talking now? `m_IsReadOnly` or `m_version`? 

- m_IsReadOnly doesn't exist in Core as we don't need it and it is marked in at least netfx >= 461 as optional.
- m_version should always be in the netfx blob. Why shouldn't it?